### ISSUE TITLE
Subscribe page styling

### ISSUE THREAD

**Issue by bramley** [phpList/phplist3#835]
This earlier commit https://github.com/phpList/phplist-ui-bootlist/commit/990bd02f12f8c0fb1985b012168147df9a2c00df removed the `invisible` class from the body element of the subscribe page.
Not sure why this was done but a side effect was to stop some styling being applied in the js function applyCustomFormatting() in file phplist_ui_bootlist.js

```
/* subscribe page */
$('body.fixed .required').addClass('text-danger');
$('body.fixed ul.list').addClass('list-unstyled');
$('body.fixed p.information').addClass('text-info text-underline').wrapInner('<big />');
$('body.fixed table:first-of-type').addClass('jumbotron'); 
$('body.fixed div.attributename').wrapInner('<big />');
$('body.fixed li.list').each(function(){
	$(this).addClass('form-group');
	var iname = $(this).find('input').attr('name'); 
	$(this).find('input').attr('id',iname);
	$(this).find('input').nextUntil('listdescription').wrapAll('<div class="btn-group" />');
	$(this).find('b').before('<label for="'+iname+'" class="btn btn-default"><span class="[ glyphicon glyphicon-ok ]"></span><span>&nbsp;</span></label>').wrap('<label for="'+iname+'"  class="btn btn-default active" />');
	});
```
The function is called only when the `body` element has the `invisible` class.
In particular it stopped the captions for mandatory fields from being displayed in red, which made the explanatory text confusing. 
`    $strRequired            = 'required fields are marked red';`

This was raised in the user forum https://discuss.phplist.org/t/subscribe-form-says-mandatory-fields-marked-red-but-no-red-is-visible/7892

**Comment by michield** [comment]
Hmm, yes, we should put that class back. I'm not sure what I tried to solve there, but I think it was that the page was empty when loading failed, which was blocking debugging. However, the invisible class is there, afaict, to have a better UX on page rendering, when the JS is loaded after the CSS.

**Comment by michield** [comment]
Should be fixed in #837
